src/main/java/de/retest/recheck/FileNamerStrategy.java

```diff
@@ -4,6 +4,10 @@
 
 public interface FileNamerStrategy {
 
+	public static final String RECHECK_FILE_EXTENSION = "recheck";
+	public static final String RECHECK_FOLDER_PROPERTY = "de.retest.RecheckFolder";
```

Comment:
Isn't this `de.retest.recheckDir`? At least that's how it is defined in `retest-core` (`CheckAccessFileImpl#RECHECK_DIR_PROP`).

Reply:
True. But this isn't a breaking change yet... only once we delete the other one.